poetry.lock

```diff
@@ -1185,6 +1187,7 @@ typing-extensions = ">=3.10"
 
 [package.extras]
 dmypy = ["psutil (>=4.0)"]
+install-types = ["pip"]
```

Comment:
Any idea what this is about?

Reply:
Mypy now declares a new package extra called `install-types`, see 
https://github.com/python/mypy/blob/bcf60ac7b7aba645bdbb2bf803d43c51f9346197/setup.py#L224
and https://github.com/python/mypy/pull/13739